browser-test/src/program_deep_link.test.ts

```diff
@@ -46,7 +46,7 @@ describe('navigating to a deep link', () => {
     // Assert
     await page.click('#continue-application-button')
     expect(await page.innerText('.cf-applicant-question-text')).toEqual(
```

Comment:
blocking: How about `toContain(questionText)`? so we can reuse the `questionText` variable

Reply:
Done.